_episodes/03-ncbi-sra.md

```diff
@@ -1,17 +1,22 @@
 ---
 title: "Examining Data on the NCBI SRA Database"
-teaching: 0
-exercises: 0
+teaching: 20
+exercises: 10
 questions:
-- "Key question"
+- "How to work with public data in the NCBI SRA"
 objectives:
-- "First objective."  
+- "Be aware that public genomic data is available"
+- "Understand how to access and download this data"  
 keypoints:
-- "First key point."
+- ""
 ---
 
+In our experiments we're usually generating our own genomic data, but many types of analyses use reference data or you may want to use it to compare your results or annotate your data with publicly available data. You may also want to do a full project or set of analyses using publicly available data. This data is a great, and essential, resource for genomic data analysis.
+
+There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) and [European Nucleotide Archive < EMBL-EBI](https://www.ebi.ac.uk/ena). In this lesson we're working with the NCBI database, but the general process is the same for any database.
```

Comment:
Should EMBL-EBI be in parens for similarity with how NCBI is referenced in the previous line?